problem80.rb

```diff
@@ -0,0 +1,24 @@
+# Transcribe the given DNA strand into corresponding mRNA - a type of RNA, that will be formed from DNA
+#  after transcription. DNA has the bases A, T, G and C, while RNA converts to U, A, C and G respectively.
+# Examples
+# dnaToRna("ATTAGCGCGATATACGCGTAC") ➞ "UAAUCGCGCUAUAUGCGCAUG" dnaToRna("CGATATA") ➞ "GCUAUAU"
+# dnaToRna("GTCATACGACGTA") ➞ "CAGUAUGCUGCAU"
+def dnaToRna(str)
+  arr = str.split("")
+  new_str = ""
+  arr.each do |x|
```

Comment:
You must try with this code instead of this piece of code:
`  arr.map {|a| a=="A" ? new_str << ("U") : a == "T" ? new_str << ("A") : a == "G" ? new_str << ("C") : new_str << ("G")}
`